Tasks/Dubinin/Task2/Task2.py

```diff
@@ -0,0 +1,62 @@
+import datetime
+
+choice=int(input('Please choose 1 - computer time, 2 - user time: '))
+
+if choice==1:
+   dt_n = datetime.datetime.now()
+   t_n = dt_n.time()
+   t_n = t_n.strftime('%H:%M')
+else:
+    t_n = input("Please enter your time in HH:MM format: ")
+
+l = t_n.split(':')
```

Comment:
you should at least check for this symbol before splitting, validation is an important part of any solution, -1 point for the unwise choice